afnexus/raid-website
# fix(images): Fix image issues, support MDX

## Description
- Add next-mdx-remote
- Remove markdown-it
- Add global alias for image loading
- Use next-images by default for MDX images
- Fix background image source


## Expected behaviour
- Images load properly and consistently

## Current behaviour
- Post images loading inconsistently from external sources
- Background image fails to load after refreshing in non-index page.

## Steps to test
1. `yarn build`
2. `yarn start`
3. make sure images are working in blog and events posts


## Linked PRs / Issue

fixes #16
